features/promotion/applying_catalog_promotions/applying_catalog_promotions_for_product_created_in_catalog_promotions_scope.feature

```diff
@@ -0,0 +1,32 @@
+@applying_catalog_promotions
+Feature: Applying catalog promotions for product created in catalog promotions scope
+    In order to have proper discounts on new products that is created in catalog promotion
+    As a Store Owner
+    I want to have catalog promotion applied on new products
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store classifies its products as "Clothes" and "Dishes"
+        And I am logged in as an administrator
+
+    @ui @javascript @no-api
+    Scenario: Applying catalog promotion on newly created simple product
```

Comment:
I'm wondering if we should cover product creation without any variant case as well